tests/multinode.rs

```diff
@@ -656,7 +656,7 @@ fn retry_get_balance(
     bob_pubkey: &PublicKey,
     expected: Option<i64>,
 ) -> Option<i64> {
-    const LAST: usize = 20;
+    const LAST: usize = 30;
```

Comment:
Why did we need more?

Reply:
With more nodes, it takes more time to synchronize. It's giving more opportunity for poll_get_balance()

Reply:
👍